day6 array methods/1.assignment array methods.js

```diff
@@ -0,0 +1,33 @@
+// filter by name
+
+const filterByName = (firstName) => {
+    return objectArray.filter(u => u.first_name===firstName);
+}
+console.log("print array filter by first name : ", filterByName("Townie"));
+
+// map by email
+const mapByEmail = () => {
+    return objectArray.map(u => u.email);
```

Comment:
we can use shorten syntax whenever possible

Reply:
got this POV of writing code, Thanks will keep in mind further.